unity/EditorPlugin/RiderPathLocator.cs

```diff
@@ -279,6 +279,8 @@ private static string[] CollectPathsFromToolbox(string toolboxRiderRootPath, str
       return paths;
     }
 
+#pragma warning disable 0649
```

Comment:
I think it would be nice to have an explanation why the warning is disabled (I've never been to that part, so if that's obvious to anyone working on that, please ignore the request :))